packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -390,3 +401,38 @@ extension DiffTreeTypeExtension on DiffTreeType {
     }
   }
 }
+
+class SingleJsonFileHeader extends DevToolsFileHeader {
```

Comment:
I don't think these need to be in their own classes. Just add the `areaPaneHeader` widget directly and you can add helpers `_generateSingleFileHeaderText` and `_generateDualFileHeaderText`